cleanlab/datalab/internal/issue_manager/underperf_group.py

```diff
@@ -126,24 +127,84 @@ def set_knn_graph(self, features: npt.NDArray, find_issues_kwargs: Dict) -> csr_
             except NotFittedError:
                 knn.fit(features)
 
-            knn_graph = knn.kneighbors_graph(mode="distance", n_neighbors=features.shape[0] - 1)
-        return knn_graph
+            knn_graph = knn.kneighbors_graph(mode="distance")
+            symmetric_knn_graph = self._symmetrize_knn_graph(knn_graph)
+        return symmetric_knn_graph, knn_graph
```

Comment:
Don't change the return value of this method `set_knn_graph`. It should just return the `knn_graph`.

It should not be responsible for the symmetrization effort because it's sometimes an unnecessary step.

```suggestion
        return knn_graph
```